conventional_pre_commit/format.py

```diff
@@ -138,9 +138,9 @@ def r_scope(self):
                 return scope_pattern
 
         if self.scope_optional:
-            return r"(\([\w \/:,-]+\))?"
+            return r"(\([\w \/:,-\.]+\))?"
         else:
-            return r"(\([\w \/:,-]+\))"
+            return r"(\([\w \/:,-\.]+\))"
```

Comment:
What do you think about pulling out `escaped_delimiters` from the `if` block above, to be able to reuse it here? I guess I didn't realize we listed these in multiple places, probably a good idea to have them once.